initfiles/bin/init_thorslave

```diff
@@ -33,11 +33,23 @@ slavename=thorslave_${hpcc_compname}
 
 stop_slaves()
 {
-    killall -0 $slavename > /dev/null 2>&1
-    if [[ $? -eq 0 ]];then
-        log "killing slaves"
-        killall -9 $slavename > /dev/null 2>&1
+    local timer=15
+    local isAlive=0
+
+    log "Attempting to kill $slavename with SIGTERM"
+    killall -SIGTERM $slavename > /dev/null 2>&1
```

Comment:
@richardkchapman went ahead and changed the style to use the verbose name like you recommended. I'll start doing this elsewhere in the init scripts as I come across it.